package.json

```diff
@@ -21,6 +21,15 @@
   "bugs": {
     "url": "http://github.com/elastic/kibana/issues"
   },
+  "pre-commit": [
```

Comment:
What is this config for? We already use husky to call `npm run precommit`, do we need both of these?

Reply:
Ah OK, no need for this then.